src/main/java/org/folio/inventory/domain/items/EffectiveCallNumberComponents.java

```diff
@@ -6,11 +6,14 @@
     private String callNumber;
     private String prefix;
     private String suffix;
+    private String typeId;
 
-    public EffectiveCallNumberComponents(String callNumber, String prefix, String suffix) {
+    public EffectiveCallNumberComponents(String callNumber, String prefix,
+                                         String suffix, String typeId) {
```

Comment:
I personally dislike this alignment of the second line of parameters, because it sacrifices so much horizontal space (which is what wrapping cares about) and is harder to maintain.

I’m aware IntelliJ does this by default, and I haven’t figured out how to change that.

What are other folks thoughts?

(I’m not saying you need to change this specific case, it’s just another vagary in code formatting)

Reply:
I don't like horizontal alignment as well. If we change the name, now we need to realign. I prefer 4 space indentation on the next line.

This is the approach Google style uses: [4.5.2 Indent continuation lines at least +4 spaces](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent). They do have a section on horizontal alignment: [4.6.3 Horizontal alignment: never required](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment). Note their "tip".

Reply:
Yes, I don't like it too.